app/electron/main.ts

```diff
@@ -1,4 +1,4 @@
-import { ChildProcessWithoutNullStreams, spawn } from 'child_process';
+import { ChildProcessWithoutNullStreams, exec,spawn } from 'child_process';
```

Comment:
Nit: This is missing a space after the new coma.
Maybe we should add prettier to the app as well.

Reply:
Oh. Thanks. 

Prettier was working for the app previously... will look into it.

Reply:
CI wasn't checking it because that is done by the frontend workflow, and this PR only changed the app. So I fixed that.

prettier is broken when using windows git, because it reformats everything with line endings. I'll leave that issue for another day outside this PR.